### DISCUSSION TITLE
ストックした記事の管理をもう少し手軽にできるようにして欲しい

### DISCUSSION THREAD

**Discussion by Mijikko** [increments/qiita-discussions#199] [要望・アイデア・不具合]
現状ではストックした記事のカテゴリ変更・ストックから削除をする場合は、1つ1つ選択して個別に管理しなければなりません。
また、記事をストックから削除する場合はストックボタンを押してチェックボックスを外す必要があります。

### 以下、要望
- ストックした記事をチェックボックスなどでまとめて選択できるようにして、まとめて管理できるようにして欲しい
- ストックした記事を削除するための専用のボタンを用意するなどして、もう少し直感的に削除できるようにして欲しい

**Comment by kskwtnk** [comment]
@MijinkoSD 
ご意見ありがとうございます！

https://github.com/increments/qiita-discussions/discussions/65 こちらのDiscussionをはじめ、ストック関連ではたくさんのご要望をいただいています
次のプロジェクトのフェーズでストックは是非改善したいと思っていますので、今しばらくお待ちください 🙏
